docs/djdknativedigest.md

```diff
@@ -23,7 +23,7 @@
 
 # -Djdk.nativeDigest
 
-This option enables or disables OpenSSL native cryptographic support for the Digest algorithm.
+This option enables or disables OpenSSL native cryptographic support for the MD5, SHA-224, SHA-256, SHA-384, and SHA-512 algorithms.
```

Comment:
It would make more sense to say `for the MD5, SHA-224, SHA-256, SHA-384, and SHA-512 digest algorithms`